src/content/1.7/development/components/back-office-help-sidebar.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Back Office Help sidebar
+menuTitle: Back Office help sidebar
+---
+
+# The Back Office Help sidebar
+
+Back Office controller return Back Office HTML pages.
+
+Most of these pages Back Office page have a common HTML structure:
```

Comment:
```suggestion
Most of these Back Office pages have a common HTML structure:
```

Reply:
Bad writing 😞  I feel ashamed

Thank you for the proof reading